primitives/src/v2/mod.rs

```diff
@@ -124,7 +129,9 @@ impl MallocSizeOf for ValidatorId {
 }
 
 /// Index of the validator is used as a lightweight replacement of the `ValidatorId` when appropriate.
-#[derive(Eq, Ord, PartialEq, PartialOrd, Copy, Clone, Encode, Decode, TypeInfo, RuntimeDebug)]
+#[derive(
+	Eq, Ord, PartialEq, PartialOrd, Copy, Clone, Encode, Decode, TypeInfo, RuntimeDebug, Default,
```

Comment:
Why did you add `Default`? We're trying to be very strict with types, especially in the runtime. It's not clear to me what `ValidatorIndex(0)` being `default` actually means.

Reply:
Added it when I was trying to derive Default for `IndexedVec`. Unnecessary now I implemented a default.